### PR TITLE
[FEATURE] What Goes Where and Style Guide

### PR DESCRIPTION
**Pull request by mabolek**
* Initial configuration
* What Goes Where section
* Style Guide section

### CODE REVIEW

File: Documentation/Index.rst [hunk 1]
```diff
@@ -1,14 +1,14 @@
 .. include:: /Includes.rst.txt
 
-=========
-<project>
-=========
+===========================
+Content and Marketing Guide
+===========================
 
 :Extension key:
```
Comment (alexander-nitsche) [inline Documentation/Index.rst:7]: For standalone guides the "Extension key" attribute should be removed. It becomes relevant for TYPO3 extension manuals only.

File: Documentation/Index.rst [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-   <extension-key>
+   guide_contentandmarketing
 
 :Package name:
```
Comment (alexander-nitsche) [inline Documentation/Index.rst:10]: For standalone guides the "Package name" attribute should be removed. It becomes relevant for TYPO3 extension and Composer package manuals only.

**Review by alexander-nitsche — COMMENTED** [review]
It would be interesting to compare this guide with the existing ones

- [how to document guide](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/Index.html) and 
- [openstrategypartners' writing guide](https://guides.openstrategypartners.com/typo3-guide/)

and to show which guide should be used when, as is done for example in the abstract of  https://docs.typo3.org/m/typo3/reference-typoscript/main/en-us/ for the different TypoScript guides.

File: Documentation/Settings.cfg [hunk 3]
```diff
@@ -3,25 +3,25 @@
...
+github_repository    = https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-ContentAndMarketing
+github_branch        = main
 
 # Footer links
-project_home         = <project-home>
-project_contact      = <project-contact>
-project_repository   = <project-repository>
-project_issues       = <project-issues>
-project_discussions  = <project-discussions>
+project_home         = https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-ContentAndMarketing
```
Comment (alexander-nitsche) [inline Documentation/Settings.cfg:18]: Missed this one: The project home of a standalone manual should be the docs.typo3.org URL:
```
project_home = https://docs.typo3.org/m/typo3/guide-contentandmarketing/main/en-us/
```
See [here](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#footer-links) for further details.

File: Documentation/Settings.cfg [hunk 4]
```diff
@@ -3,25 +3,25 @@
...
+version     = 1.0
+release     = 1.0
+copyright   = TYPO3 Project Contributors
 
 [html_theme_options]
 
 # "Edit on GitHub" button
-github_repository    = <github-repository>
-github_branch        = <github-branch>
+github_repository    = https://github.com/TYPO3-Documentation/TYPO3CMS-Guide-ContentAndMarketing
```
Comment (alexander-nitsche) [inline Documentation/Settings.cfg:14]: Missed this one: The repository must not include the protocol and domain, but only 
```
github_repository = TYPO3-Documentation/TYPO3CMS-Guide-ContentAndMarketing
```
See [here](https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/GeneralConventions/FileStructure.html#github-workflow) for further details.

**Review by alexander-nitsche — COMMENTED** [review]
Missed two places.

**Review by linawolf — APPROVED** [review]

**Pull request merged**